airflow/providers/databricks/operators/databricks.py

```diff
@@ -24,6 +24,8 @@
 from logging import Logger
 from typing import TYPE_CHECKING, Any, Sequence
 
+from databricks.sdk.service import jobs as j
```

Comment:
```suggestion
from databricks.sdk.service import jobs```

Looks to me more readable, wdyt?

Reply:
resolved